daft/expressions/expressions.py

```diff
@@ -4243,7 +4243,7 @@ def days(self) -> Expression:
         """Partitioning Transform that returns the number of days since epoch (1970-01-01).
```

Comment:
Minor, but might be worth updating the description here will some context on the choice of type.

Reply:
Added an explanation below!